README.md

```diff
@@ -23,70 +24,61 @@ A pre-built OpenAPI Specification is available [here](https://aerospike.github.i
 - [FAQ](#faq)
 
 ## Getting Started
-
+Aerospike Backup Service reads configurations from a YAML file provided when the service is launched. See [Run](#run) for specific syntax.
 A sample configuration file and docker-compose script will help you get started testing the service.
-Follow the [instructions](./docker-compose) to set up your test environment using docker-compose.
+Follow the [docker-compose instructions](./docker-compose) to set up your test environment.
 
-Linux installation packages are available under the [releases](https://github.com/aerospike/aerospike-backup-service/releases).
+Linux installation packages are available under [releases](https://github.com/aerospike/aerospike-backup-service/releases).
 
 ## User Guide
 
 ### Entities
```

Comment:
Entities are types of configuration components, not endpoints. The further documentation contains an incorrect explanation of them.

Reply:
Can you point out the incorrect explanation? For example, can we say that an entity "includes" endpoints?

Reply:
An example for an entity would be backup-policy, backup-routine etc'. There are endpoints to operate entities. So instead of saying "Cluster configuration endpoints establish connections to Aerospike clusters." we should use something like "Cluster configuration entites denote the configuration properties needed to establish connections to Aerospike clusters."